### ISSUE TITLE
Kolbot Wont Select Character

### ISSUE THREAD

**Issue by Spink2019** [kolton/d2bot-with-kolbot#2932]
After two months of no issues one of my follow bots does not work properly. D2 loads, signs in, gets stuck on select character. Delete profile - still doesnt work. Delete char config - still doesnt work. Fresh install completely - still doesnt work. 

Any advice?

**Comment by gtoilet** [comment]
profile toon name needs to be case specific
should work fine stock lod install with bot from this github works just fine
installed on win 10 pc just last week with no issue

**Comment by Spink2019** [comment]
> profile toon name needs to be case specific
> should work fine stock lod install with bot from this github works just fine
> installed on win 10 pc just last week with no issue

It should work. But it randomly stopped working. Char with same settings did 1000+ runs. I renamed my actual D2 char. That worked. lol

**Comment by gtoilet** [comment]
yes if you havent messed with the oog or the entry points case numbers
then the only places it checks to determine who to select at char select screen
are the name you put is in the char profile you set up bot
